doc/modules/grid_search.rst

```diff
@@ -88,6 +88,12 @@ evaluated and the best combination is retained.
       for an example of :class:`GridSearchCV` being used to evaluate multiple
       metrics simultaneously.
 
+    - See :ref:`sphx_glr_auto_examples_model_selection_plot_grid_search_refit_callable.py` for an
+      example of using ``refit=callable`` interface in :class:`GridSearchCV`. The example shows this
+      interface adds certain amount of flexibility in identifying the "best" estimator. The function
```

Comment:
I don't think the sentence 'The function passed to parameter ``refit`` incorporate of which metric(s) to optimise. ' is necessary. I think you can drop the following sentence too.